docs/resources/driver.md

```diff
@@ -0,0 +1,217 @@
+
+### Optional
+
+- `timeouts` (Block, Optional) (see [below for nested schema](#nestedblock--timeouts))
+
+<a id="nestedblock--metadata"></a>
+### Nested Schema for `metadata`
+
+***Required***
+
+- `name` (String) name of the resource
+- `project` (String) Project of the resource
+
+***Optional***
+
+- `description` (String) description of the resource
+
+
+<a id="nestedblock--spec"></a>
+### Nested Schema for `spec`
+
+***Required***
+
+- `config` (Block List, Max: 1) Driver configuration (see [below for nested schema](#nestedblock--spec--config))
+
+***Optional***
+
+- `sharing` (Block List, Max: 1) Sharing with other projects (see [below for nested schema](#nestedblock--spec--sharing))
+
+<a id="nestedblock--spec--config"></a>
+### Nested Schema for `spec.config`
+
+***Required***
+
+- `type` (String) Specify the type of driver, Options are container, http
+- `container` (Block List, Max: 1) Specify the container driver config if type is container (see [below for nested schema](#nestedblock--spec--config--container))
+- `http` (Block List, Max: 1) Specify the http driver config if type is http (see [below for nested schema](#nestedblock--spec--config--http))
+
+***Optional***
+
+- `max_retry_count` (Number) Specify the max retry count
+- `success_condition` (String) Specify the success condition
+- `timeout_seconds` (Number) Specify the timeout in seconds
+
+<a id="nestedblock--spec--config--container"></a>
+### Nested Schema for `spec.config.container`
+
+***Required***
+
+- `image` (String) Specify the container image for the driver
+
+***Optional***
+
+- `arguments` (List of String) Specify the set of arguments to be passed
+- `commands` (List of String) Specify the set of commands to be executed
+- `cpu_limit_milli` (String) Specify the cpu limit in milli
+- `env_vars` (Map of String) Specify the environment variables to be set in key,value pair
+- `files` (Map of String) Specify the file data
+- `image_pull_credentials` (Block List, Max: 1) Specify the credentials for the registry to pull image from (see [below for nested schema](#nestedblock--spec--config--container--image_pull_credentials))
+- `kube_config_options` (Block List, Max: 1) Specify the kube config options (see [below for nested schema](#nestedblock--spec--config--container--kube_config_options))
+- `kube_options` (Block List, Max: 1) Specify the kube options (see [below for nested schema](#nestedblock--spec--config--container--kube_options))
+- `memory_limit_mb` (String) Specify the memory limit to be allocated in MB
+- `volume_options` (Block List, Max: 1) Specify the container driver volume options (see [below for nested schema](#nestedblock--spec--config--container--volume_options))
+- `working_dir_path` (String) Specify the working directory path
+
+<a id="nestedblock--spec--config--container--image_pull_credentials"></a>
+### Nested Schema for `spec.config.container.working_dir_path`
+
+***Optional***
```

Comment:
this should be ***Required***?

Reply:
removed driver.md as of now, currently it is in development